### ISSUE TITLE
tf.nightly-gpu will create "nan" in  next batch train value

### ISSUE THREAD

**Issue by Q82822** [tensorflow/tensorflow#22108]
------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: 
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:Linux Ubuntu 16.04
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: 1.11.0   (use tf-night-gpu)
- **Python version**:2.7
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**: 9.0 /7.1
- **GPU model and memory**:
- **Exact command to reproduce**:

### Describe the problem
I installed  tf-night-gpu, and my` tf.__version__` was `1.11.0-dev20180905`
it only use `tf-night-gpu` can create  the `tf.keras.applications.InceptionResNetV2` model so i can't without. my problem is the  `tf-night-gpu` will create `nan`  in  next batch model output 

### Source code / logs
in first batch train the
```
('keras.layers.Embedding_input: ', <tf.Tensor: id=4586198, shape=(64, 1), dtype=int32, numpy=
array([[3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3]], dtype=int32)>)
('Embedding_output:  ', <tf.Tensor: id=4586379, shape=(64, 1, 256), dtype=float32, numpy=
array([[[ 0.01097491, -0.02041122,  0.04823625, ..., -0.0155656 ,
          0.01521399, -0.0496642 ]],

       [[ 0.01097491, -0.02041122,  0.04823625, ..., -0.0155656 ,
          0.01521399, -0.0496642 ]],

       [[ 0.01097491, -0.02041122,  0.04823625, ..., -0.0155656 ,
          0.01521399, -0.0496642 ]],

       ...,

       [[ 0.01097491, -0.02041122,  0.04823625, ..., -0.0155656 ,
          0.01521399, -0.0496642 ]],

       [[ 0.01097491, -0.02041122,  0.04823625, ..., -0.0155656 ,
          0.01521399, -0.0496642 ]],

       [[ 0.01097491, -0.02041122,  0.04823625, ..., -0.0155656 ,
          0.01521399, -0.0496642 ]]], dtype=float32)>)
```
the next batch train , have same `keras.layers.Embedding_input: `
```
`Epoch 1 Batch 0 Loss 2.0415
(''keras.layers.Embedding_input:: ', <tf.Tensor: id=4607289, shape=(64, 1), dtype=int32, numpy=
array([[3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3],
       [3]], dtype=int32)>)
(''keras.layers.Embedding_output::  ', <tf.Tensor: id=4607374, shape=(64, 1, 256), dtype=float32, numpy=
array([[[nan, nan, nan, ..., nan, nan, nan]],

       [[nan, nan, nan, ..., nan, nan, nan]],

       [[nan, nan, nan, ..., nan, nan, nan]],


       [[nan, nan, nan, ..., nan, nan, nan]],

       [[nan, nan, nan, ..., nan, nan, nan]],

       [[nan, nan, nan, ..., nan, nan, nan]]], dtype=float32)>)`
```
and "nan" forever.....

**Comment by omalleyt12** [comment]
@Q82822 could you please give a minimal code example that reproduces the issue?

**Comment by Q82822** [comment]
the minimal code at  here , it is just part of my project code. the inpout is word_index.
the batch size = 64 and use tf-nightly-gpu.
in first batch size was worked, but in next batch train will be "nan".
```
class  RNN_Decoder(tf.keras.Model):
    def __init__(self, embedding_dim, units, vocab_size):
        super(RNN_Decoder, self).__init__()
        self.units = units
        self.embedding = tf.keras.layers.Embedding(vocab_size, embedding_dim)
    def call(self, x, features, hidden):
        print('embedding_input:  ', x)
        x = self.embedding(x)
        print('embedding_output:  ', x)

decoder = RNN_Decoder(embedding_dim, units, vocab_size)

for epoch in range(20):
    hidden = decoder.reset_state(batch_size=64)
        with tf.GradientTape() as tape:
            for i in range(1, target.shape[1])
            predictions, hidden, _ = decoder(dec_input, features, hidden)
```

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 60 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 75 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by chunduriv** [comment]
@Q82822,

Sorry for the late response. Is this still an issue for you? 

We see that you are using an old version of tensorflow (1.x) which is not actively supported, We recommend that you upgrade to 2.8.0 and let us know if the issue still persists in newer versions .Thanks!
